tests/sftp/triggers/test_sftp.py

```diff
@@ -144,3 +153,21 @@ async def test_sftp_trigger_run_trigger_failure_state(self, mock_get_file_by_pat
             actual_event = await generator.asend(None)
 
             assert TriggerEvent(expected_event) == actual_event
+
+    @pytest.mark.asyncio
+    @mock.patch("astronomer.providers.sftp.hooks.sftp.SFTPHookAsync.get_files_by_pattern")
+    async def test_sftp_trigger_run_airflow_exception(self, mock_get_files_by_pattern):
+        """
+        Assert that a the task does not complete if the hook raises an AirflowException,
+        indicating that the task needs to be deferred
+        """
+        mock_get_files_by_pattern.side_effect = AirflowException("No files at path found...")
```

Comment:
*nitpick*

it seems this might not be a valid message for 

```python
            raise AirflowException(f"No files at path {path} found...")
```

Maybe something like `No files at path swh path found...` would be better, but it won't affect the test.

Reply:
`AirflowException("No files at path /test/path/ found...")`